src/users/dto/create-user.dto.ts

```diff
@@ -25,5 +25,6 @@ export class CreateUserDto {
   surname: string;
   @IsInt()
   @IsPositive()
+  @IsNotEmpty()
   phoneNumber: number;
```

Comment:
https://stackoverflow.com/questions/8284647/how-to-store-phone-numbers-on-mysql-databases maybe you can find something useful there